vhd/release-notes/aks-windows-2019-containerd/2019-datacenter-core-ctrd-17763.1637.201210.txt

```diff
@@ -0,0 +1,118 @@
+﻿Build Number: 20201210.5

```

Comment:
should this be apart of this PR?

Reply:
I meant to include it since we're adding a reference to that VHD in azenvtypes.

Reply:
see that now!